fe/fe-core/src/main/java/com/starrocks/analysis/ColumnDef.java

```diff
@@ -109,6 +111,10 @@ public boolean isAllowNull() {
         return isAllowNull;
     }
 
+    public void setIsAllowNullForPrimaryKeySql() {
```

Comment:
Need to comment why PrimaryKey is special.

Reply:
added.